linkage-monitor/action.yml

```diff
@@ -7,7 +7,7 @@ runs:
       # scripts/release.sh updates the version part in the URL
       run: |
         curl --output /tmp/linkage-monitor.jar \
-        "https://storage.googleapis.com/cloud-opensource-java-linkage-monitor/linkage-monitor-1.5.5-SNAPSHOT-all-deps.jar"
+        "https://storage.googleapis.com/cloud-opensource-java-linkage-monitor/linkage-monitor-1.5.5-all-deps.jar"
```

Comment:
This is the what I wanted for this pull request. This file at "v1.5.5-dependencies" tag should download the file. I confirmed that both URLs work good:

- https://storage.googleapis.com/cloud-opensource-java-linkage-monitor/linkage-monitor-latest-all-deps.jar
- https://storage.googleapis.com/cloud-opensource-java-linkage-monitor/linkage-monitor-1.5.5-all-deps.jar